cloudnet/src/main/resources/lang/dutch.properties

```diff
@@ -28,243 +28,243 @@ cloudnet-init-setup-tasks-should-install-server=Wilt u standaard een lobby maken
+cloudnet-pre-unload-module=Modul %module_group%:%module_name%:%module_version% wird entfernt...
+cloudnet-post-unload-module=Modul %module_group%:%module_name%:%module_version% wurde erfolgreich entfernt!
+cloudnet-pre-install-dependency-module=Installiere Abhängigkeit %group%:%name%:%version% für das Modul %module_group%:%module_name%:%module_version%...
+cloudnet-post-install-dependency-module=Abhängigkeit %group%:%name%:%version% für das Modul %module_group%:%module_name%:%module_version% wurde erfolgreich installiert!
 #
 # Console animations
 #
-ca-question-list-explain = Gebruik de pijltjestoetsen om door alle mogelijke waarden te scrollen (indien ingesteld). Als er een aanbevolen waarde is, wordt deze naar de opdrachtregel geschreven wanneer de vraag wordt weergegeven
-ca-question-list-cancel = Dit proces kan op elk moment worden beëindigd met "cancel"!
-ca-question-list-canceled = Het proces is geannuleerd!
-ca-question-list-question-list = Voer een van de volgende waarden in: %values%
-ca-question-list-invalid-default = Voer een geldige waarde in!
-ca-question-list-invalid-uuid = Voer een geldige UUID in!
-ca-question-list-invalid-service-version = Deze versie bestaat niet!
-ca-question-list-invalid-int = Voer een geldig geheel getal in!
-ca-question-list-invalid-int-range = Voer een geldig geheel getal in tussen %min% en %max%!
-ca-question-list-invalid-double = Voer een geldig decimaal getal in!
-ca-question-list-invalid-url = Voer een correcte URL in!
-ca-question-list-invalid-boolean = Voer "%true%" of "%false%" in!
-ca-vragenlijst-boolean-true = ja
-ca-vragenlijst-boolean-false = nee
-ca-question-list-invalid-template = Voer een geldige sjabloon in het volgende formaat in: opslag: prefix / naam (opslag is optioneel)
-ca-question-list-template-invalid-storage = Deze opslag bestaat niet!
-ca-question-list-possible-answers-list = Beschikbare antwoorden: %values%
+ca-question-list-explain=Gebruik de pijltjestoetsen om door alle mogelijke waarden te scrollen (indien ingesteld). Als er een aanbevolen waarde is, wordt deze naar de opdrachtregel geschreven wanneer de vraag wordt weergegeven
+ca-question-list-cancel=Dit proces kan op elk moment worden beëindigd met "cancel"!
+ca-question-list-canceled=Het proces is geannuleerd!
+ca-question-list-question-list=Voer een van de volgende waarden in: %values%
+ca-question-list-invalid-default=Voer een geldige waarde in!
+ca-question-list-invalid-uuid=Voer een geldige UUID in!
+ca-question-list-invalid-service-version=Deze versie bestaat niet!
+ca-question-list-invalid-int=Voer een geldig geheel getal in!
+ca-question-list-invalid-int-range=Voer een geldig geheel getal in tussen %min% en %max%!
+ca-question-list-invalid-double=Voer een geldig decimaal getal in!
+ca-question-list-invalid-url=Voer een correcte URL in!
+ca-question-list-invalid-boolean=Voer "%true%" of "%false%" in!
+ca-vragenlijst-boolean-true=yes
+ca-vragenlijst-boolean-false=no
```

Comment:
These are english, is this correct?

Reply:
Not only are they in english but the keys are in dutch